testing/suite/command_line.go

```diff
@@ -107,3 +109,50 @@ func (c *CommandLine) Exec() error {
 
 	return nil
 }
+
+func (c *CommandLine) ExecWithCustomPipe(w, r *os.File) error {
+	outC := make(chan string)
+	// copy the output in a separate goroutine so it won't block indefinitely
+	go func() {
+		var buf bytes.Buffer
+		io.Copy(&buf, r)
+		outC <- buf.String()
+	}()
+
+	// use command stdin
+	cmd := exec.Command(c.cmd, c.args...)
+	cmd.Env = append(os.Environ(), c.env...)
+	stdin, err := cmd.StdinPipe()
+	if err != nil {
+		c.err = err.Error()
+		return err
+	}
+
+	if c.surveyAnswer != "" {
```

Comment:
Do we need check for surveyAnswer in workload get?

Reply:
this is not just for `workload get` command. Eventually, we will be testing `workload create` and `apply` as well in e2e, so we will need a way to answer surveys